mne_connectivity/spectral/time.py

```diff
@@ -302,12 +391,42 @@ def spectral_connectivity_time(data, freqs, method='coh', average=False,
 
     # get indices of pairs of (group) regions
     if indices is None:
-        indices_use = np.tril_indices(n_signals, k=-1)
+        if multivariate_con:
+            if any(this_method in _gc_methods for this_method in method):
+                raise ValueError(
```

Comment:
Can you add a unit-test for this case?

Reply:
Done! https://github.com/tsbinns/mne-connectivity/blob/894c42e59cef9f97aa07a7a4f71cef756512c50c/mne_connectivity/spectral/tests/test_spectral.py#L1132-L1135